src/Randomizer.SMZ3.Tracking/AutoTracking/GameService.cs

```diff
@@ -227,27 +242,20 @@ public bool TryFillArrows()
         /// <returns>False if it is currently unable to give rupees to the player</returns>
         public bool TryFillRupees()
         {
-            if (_autoTracker == null || !_autoTracker.IsConnected || _autoTracker.CurrentGame != Game.Zelda || !_autoTracker.IsInSMZ3)
+            if (AutoTracker == null || !AutoTracker.IsConnected || AutoTracker.CurrentGame != Game.Zelda || !AutoTracker.IsInSMZ3)
             {
                 return false;
             }
 
-            List<byte> bytes = BitConverter.GetBytes((short)2000).ToList();
-
-            if (!BitConverter.IsLittleEndian)
-            {
-                bytes.Reverse();
-            }
-
             // Writing the target value to $7EF360 makes the rupee count start counting toward it.
             // Writing the target value to $7EF362 immediately sets the rupee count, but then it starts counting back toward where it was.
             // Writing the target value to both locations immediately sets the rupee count and keeps it there.
-            _autoTracker.WriteToMemory(new EmulatorAction()
+            AutoTracker.WriteToMemory(new EmulatorAction()
             {
                 Type = EmulatorActionType.WriteBytes,
                 Domain = MemoryDomain.WRAM,
                 Address = 0x7EF360,
-                WriteValues = bytes.Concat(bytes).ToList()
+                WriteValues = Int16ToBytes(2000)
```

Comment:
I was being a little too cute here by concatenating the list to itself; we need to write the value to both `$7EF360` and `$7EF362` or the rupee count will immediately start counting back to where it was.

Reply:
Ah, thanks for catching that. I must have glossed over what you were doing there when making changes. Interestingly though, I did test it, and it just climbed from 0 to 2000 rupees.

Reply:
For the endian-ness, I don't think we really need to bother. I did a bit of searching, and it really seems like pretty much all home computers use little endian.

Reply:
I misread my own comment! Haha!

And yeah, I agree that we probably don't need to bother accounting for big-endian systems at this time.